pom.xml

```diff
@@ -24,7 +24,7 @@
     <dependency>
       <groupId>uk.gov.ons.ctp.product</groupId>
       <artifactId>collectionexercisesvc-api</artifactId>
-      <version>10.49.4</version>
+      <version>10.49.5-SNAPSHOT</version>
```

Comment:
If we'll be using Events to drive actions, could we please update the `actionSvc` POM as well?